code/models/models.js

```diff
@@ -89,3 +89,58 @@ exports.updateArticleVotes = (request) => {
     return result.rows;
   });
 };
+
+
+
+
+exports.insertComment = (request) => {
+  const articleID = request.params.articles_id;
+  const commentsReceived = request.body;
+  // construct the comment to insert:
+  let commentToInsert = {
+    //comment_id: --> to be provided by the psql query,
+    body: commentsReceived.body,
+    article_id: articleID,
+    author: commentsReceived.username,
+    votes: 0,
+    // created_at: --> psql will provide by using "DEFAULT NOW()",
+  };
+  let queryString = `
+  INSERT INTO comments
+  (body, author, article_id, votes) 
+  VALUES
+  ('${commentToInsert.body}','${commentToInsert.author}',${commentToInsert.article_id},${commentToInsert.votes})
```

Comment:
This is SQL injection. Please use the $ syntax to prevent this.

Reply:
Done - chaged to the $ syntax and added a valueArray into the db.query as 2nd argument